gemfiles/rails4.2.gemfile

```diff
@@ -4,5 +4,6 @@ source "https://rubygems.org"
 
 gem "rubocop", "~> 0.52.0"
 gem "rails", "~> 4.2.0"
+gem "mongoid", "~> 5"
```

Comment:
These files are autogenerated, you have to specify this version in `Appraisals` so that the version doesn't get overwritten when running `appraisal update`.

Reply:
Got it. Appraisals has been updated.